checker/src/org/checkerframework/checker/nullness/qual/Nullable.java

```diff
@@ -29,8 +29,6 @@
  */
 @SubtypeOf({})
 @ImplicitFor(trees = { Tree.Kind.NULL_LITERAL }, typeNames = { java.lang.Void.class })
-@DefaultFor({ DefaultLocation.LOCAL_VARIABLE, DefaultLocation.RESOURCE_VARIABLE,
```

Comment:
Now that these annotations are removed, the corresponding imports should also be removed.